public/stylesheets/game.sass

```diff
@@ -0,0 +1,24 @@
+$hand-border: 3px dashed white
```

Comment:
Need to move this into the plugin directory (and wire up the Sass processing appropriately).